libbroker/broker/internal/core_actor.hh

```diff
@@ -60,15 +60,18 @@ public:
   };
 
   /// The result of invoking a callback on a handler.
-  enum class [[nodiscard]] handler_result {
+  enum class handler_result {
     /// The callback was invoked successfully.
     ok,
     /// The callback was invoked but resulted in a terminal state.
-    disposed,
+    disconnect,
```

Comment:
Thank you because I didn't like `disposed` and was going to comment on that. :-)